backend/src/contexts/service.py

```
@@ -6,11 +6,19 @@
 from langgraph.store.base import BaseStore
 from src.utils.logger import logger
+from langgraph.store.memory import InMemoryStore
+
+IN_MEMORY_STORE = InMemoryStore()
+
 class ServiceContext:
-	
-	def __init__(self, user_id: str, store: BaseStore, checkpointer: Optional[BaseCheckpointSaver] = None):
+	def __init__(
+		self, 
+		user_id: str = None, 
```

Ruff (0.14.1)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)